SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorLayoutDetails.cpp

```diff
@@ -103,9 +104,13 @@ void FSpatialGDKEditorLayoutDetails::CustomizeDetails(IDetailLayoutBuilder& Deta
 
 FReply FSpatialGDKEditorLayoutDetails::GenerateDevAuthToken()
 {
+
+	FString Command = GetDefault<USpatialGDKSettings>()->IsRunningInChina() ? TEXT("project auth dev-auth-token create --environment cn-production --description=\"Unreal GDK Token\" --json_output") :
```

Comment:
I think I would prefer to just append the `--environment` flag if we are running in china instead of duplicating the arguments list in both cases

Reply:
very small nit: would  rename the variable to `Arguments` instead of `Command`